PyInstaller/compat.py

```diff
@@ -198,6 +198,29 @@
     modname_tkinter = 'tkinter'
 
 
+# On Windows we require pypiwin32 or pywin32-ctypes
+# -> all pyinstaller modules should use win32api from PyInstaller.compat to
+#    ensure that it can work on MSYS2 (which requires pywin32-ctypes)
+if is_win:
+    try:
+        from PyInstaller.utils.win32 import winutils
+        try:
+            pywintypes = winutils.import_pywin32_module('pywintypes', _is_venv=is_venv)
+            win32api = winutils.import_pywin32_module('win32api', _is_venv=is_venv)
+        except ImportError:
+            try:
+                from win32ctypes.pywin32 import pywintypes
+                from win32ctypes.pywin32 import win32api
+            except ImportError:
+                raise
+    except ImportError:
+        # Hack required to allow setup.py to work
```

Comment:
Please add something like "This variable is set by setup.py" (this is not hard to guess, but not obvious ;-)